### DISCUSSION TITLE
Unhandled promise rejection warning error in latest version v1.0.0.rc0

### DISCUSSION THREAD

**Discussion by OrigSorceror** [samemory/homebridge-eufy-security-control#181] [General]
(node:24662) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'indexOf' of undefined
    at EufySecurityPlatform.discoverDevices (/usr/local/lib/node_modules/homebridge-eufy-security/src/platform.ts:176:38)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at HomebridgeAPI.<anonymous> (/usr/local/lib/node_modules/homebridge-eufy-security/src/platform.ts:121:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:24662) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:24662) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by OrigSorceror** [comment]
Nevermind..  Issue resolved once i re-registered my log-in details in setup.

**Comment by lenoxys** [comment]
thanks for your feedback ! Closing this!
